### PR TITLE
[Merged by Bors] - Add mock implementation instead of panic for stack ops

### PR DESCRIPTION
**Pull request by acrrd**

### CODE REVIEW

File: discovery_engine_flutter/example/pubspec.lock [hunk 1]
```diff
@@ -12,7 +12,7 @@ packages:
     dependency: transitive
     description:
       path: async_bindgen_dart_utils
-      ref: HEAD
+      ref: fe195d99954c691c5e606ebbb3c77d371ba0e65b
```
Comment (Robert-Steiner) [inline discovery_engine_flutter/example/pubspec.lock:15]: is this always changing because we use different flutter versions locally?

**Review by Robert-Steiner — APPROVED** [review]

File: discovery_engine_flutter/example/pubspec.lock (hunk 1 above)
Comment (Robert-Steiner) [reply discovery_engine_flutter/example/pubspec.lock:15]: was discussed in the daily

Comment (acrrd) [reply discovery_engine_flutter/example/pubspec.lock:15]: I was on a branch where this wasn't already there. Now is on main so I don't expect it to change anymore.

**Comment by acrrd** [comment]
bors merge

**Pull request closed without merging**